Sources/xcodeproj/Extensions/Decodable+Dictionary.swift

```diff
@@ -8,8 +8,10 @@ extension Decodable {
     /// - Parameter jsonDictionary: json dictionary.
     /// - Throws: throws an error if the initialization fails.
     init(jsonDictionary: [String: Any]) throws {
```

Comment:
We can get rid of this extension. I searched for it in the project and is only used in a couple of tests, which can be refactored to not use it.

Reply:
I will move this file to the test folder - `Tests/xcodeprojTests / Extensions`